pylib/anki/collection.py

```diff
@@ -495,55 +495,26 @@ def find_cards(
         query: str,
         order: Union[bool, str, BuiltinSort.Kind.V] = False,
         reverse: bool = False,
-    ) -> Sequence[CardId]:
+    ) -> List[CardId]:
         """Return card ids matching the provided search.
-
         To programmatically construct a search string, see .build_search_string().
-
-        If order=True, use the sort order stored in the collection config
-        If order=False, do no ordering
-
-        If order is a string, that text is added after 'order by' in the sql statement.
-        You must add ' asc' or ' desc' to the order, as Anki will replace asc with
-        desc and vice versa when reverse is set in the collection config, eg
-        order="c.ivl asc, c.due desc".
-
-        If order is a BuiltinSort.Kind value, sort using that builtin sort, eg
-        col.find_cards("", order=BuiltinSort.Kind.CARD_DUE)
-
-        The reverse argument only applies when a BuiltinSort.Kind is provided;
-        otherwise the collection config defines whether reverse is set or not.
+        To define a sort order, see _build_sort_mode().
```

Comment:
That's tricky. On the one hand, it should be private since it returns pb stuff and is only called from inside other pylib methods. On the other hand, its documentation needs to be public.
I think I'll move the docstring back into find_cards() and just reference it in find_notes().